CHANGELOG.md

```diff
@@ -1,52 +1,96 @@
+| Story | Newsletter Subscription                                                                                                                 | [3521][]              |
+| Story | Contact Us Block                                                                                                                        | [3544][][13][]        |
+| Story | Contact Us                                                                                                                              | [3538][]              |
+| Story | Guest Checkout: check if email is associated with the account                                                                           | [3529][]              |
+| Story | Feature: Add ability to deploy Luma and Venia on single cloud env                                                                       | [11][]                |
+| Story | Add cypress test for Mini cart                                                                                                          | [3559][]              |
+| Story | Archive MFTF Repo                                                                                                                       | [39][]                |
+| Story | Homepage unification for CE and EE                                                                                                      | [3564][]              |
+| Story | Improve Version banner                                                                                                                  | [3555][]              |
+| Story | Upgrade Node 14 and Scaffold Node 16                                                                                                    | [3552][][101][]       |
+| Story | Fail build when @magento/pwa metapackage is not installed                                                                               | [3561][]              |
+| Story | Filtering Product by the custom product attribute with input type select                                                                | [15][]                |
+| Story | Do not show Ratings                                                                                                                     | [3654][]              |
+| Story | Viewing product attribute value in PDP                                                                                                  | [3624][][14][]        |
+| Story | Sorting product listing by custom attribute                                                                                             | [3619][][][]          |
+| Story | \[GraphQL] Sort products without custom attribute to end                                                                                | [18][]                |
+| Bug   | 404 routes cause error page                                                                                                             | [3542][]              |
```

Comment:
Starting here, a bunch of links aren't getting rendered correctly:

![Screen Shot 2022-01-28 at 10 46 13 AM](https://user-images.githubusercontent.com/13182778/151587357-7f15e8d4-e5dc-4bb0-afec-b92b9ab4bea9.png)